yw/Software/src/spellchecker/Checker2.java

```diff
@@ -0,0 +1,32 @@
+package spellchecker;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Checker2 {
+
+	public String check(String arr) throws IOException {
+		Read r = new Read();
+		String[] sentence = arr.split("\\s+"); 
+		List<String> suspects = r.read();  // receive suspects
```

Comment:
This variable should be renamed "stopwords"

Reply:
Thanks for pointing out, the problem has been fixed